alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -320,19 +321,45 @@ private function getIfarmEmbedCode($bundleContent)
                     </html >';
 		return $htmlDoc;
 	}
+
+	private function toAssociativeArray($input)
+	{
+		$configs = explode(",", $input);
+		$arr = array();
+		foreach($configs as $conf)
+		{
+			$obj = explode("=", $conf);
+			$key = $obj[0];
+			$value = $obj[1];
+			$arr[$key] = $value;
+		}
+		return $arr;
+	}
+
+	private function mergeVersionsParamIntoConfig()
+	{
+		//Get version from QS
+		$versions = $this->getRequestParameter(self::VERSIONS_PARAM_NAME);
```

Comment:
IF versions param was not passed you can already leave this function, no need to continue and run the perg_match

Reply:
done.